Readme tweaks

tekwiz opened the pull request:
**What is the purpose of this pull request?**

- [x] Documentation update
- [x] Bug fix
- [ ] New feature
- [ ] Other, please explain:

**What changes did you make? (provide an overview)**

Updated examples in the README to:
1. fix an error with file-type fromBuffer -- 4edad8ecea717482b460196f5de9243615ebce1e
2. make examples consistently use async/await -- 9647772869f6de9b21550ec48a1511f054476577
3. show how to use a custom error to store an error response for extraction of the body in the catch statement - e9cc7f1625d92ce211f72baf27ae212bd9e34a45

**Which issue (if any) does this pull request address?**

Fixes #1009 

**Is there anything you'd like reviewers to know?**

no

Richienb reviewed (changes requested):
The examples should assume an environment where top-level await statements are allowed.

tekwiz commented:
@Richienb I've converted the example to top-level await.  I also fixed two other minor things in the README while I was at it:  1665988e2493beac877ff9de96ecd35e1c01225c and f4aef932b5fecddae7a078d588d65063acd08c98
